keyboards/mouse/ploopyco_trackball/spi.c

```diff
@@ -0,0 +1,124 @@
+
```

Comment:
Won't bother making comments on the SPI stuff -- it should be converted over to the `spi_master` APIs instead.